catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/MetacardApplication.java

```diff
@@ -517,6 +526,112 @@ public void init() {
             return "";
         });
 
+        post("/annotations", (req, res) -> {
+            Map<String, Object> incoming = JsonFactory.create()
+                    .parser()
+                    .parseMap(req.body());
```

Comment:
Ooooh, I think in general we haven't been protecting against that in any of these endpoints.  It looks like the whole class would need to be updated.
https://github.com/codice/ddf/pull/2222/files#diff-691f835bd92e972f33f434a9c0957affL224
https://github.com/codice/ddf/pull/2222/files#diff-691f835bd92e972f33f434a9c0957affL238
https://github.com/codice/ddf/pull/2222/files#diff-691f835bd92e972f33f434a9c0957affL256
...